exercises/nucleotide-count/src/example.c

```diff
@@ -0,0 +1,44 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include "nucleotide_count.h"
+
+char *count(const char *dna_strand)
+{
+   bool invalidChar = false;
+   uint16_t index;
+   uint16_t count_a = 0;
+   uint16_t count_c = 0;
+   uint16_t count_g = 0;
+   uint16_t count_t = 0;
```

Comment:
Very very minor adjustment: maybe count names could instead be of the form `x_count`? This way `count_t` would no longer seem like a type name (`_t` suffix is often used to represent type names), yet the names would still make sense 😃